src/Assets/Bugsnag/Editor/BugsnagEditor.cs

```diff
@@ -6,12 +6,21 @@
 using BugsnagUnity;
 using UnityEditor.Callbacks;
 using System.Linq;
+using System;
 
 namespace BugsnagUnity.Editor
 {
     public class BugsnagEditor : EditorWindow
     {
 
+        private const string ANDROID_DEPS_XML = "<dependencies><androidPackages><repositories><repository>https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib</repository></repositories><androidPackage spec=\"org.jetbrains.kotlin:kotlin-stdlib:1.5.0\"></androidPackage></androidPackages></dependencies>";
+
+        private const string EDM_MENU_ITEM = "Window/Bugsnag/Enable EDM4U Support";
+
+        private static string EDMDepsFilePath = "/Bugsnag/Editor/BugsnagAndroidDependencies.xml";
+
+        private static string KotlinLibsDirPath = "/Bugsnag/Plugins/Android/Kotlin";
```

Comment:
These are always used with `Application.dataPath` - suggest prepending it here.

Reply:
You can't use Application.dataPath in an initialiser. Unity throws an error if you do.